hack/dockerfiles/gosec.Dockerfile

```diff
@@ -0,0 +1,7 @@
+# syntax=docker/dockerfile:1.3
+
+ARG GOSEC_VERSION=v2.9.1
```

Comment:
Wondering; instead of a new Dockerfile, wouldn't we be able to enable `gosec` in `golangci-lint` ? Looks like it's included in golangci-lint, but not enabled by default; https://golangci-lint.run/usage/linters/

Reply:
Yep, will come back to this. Currently working on the same in my runc PR. ;-)